acceptance_tests/features/load_seft_collection_instrument.feature

```diff
@@ -4,7 +4,19 @@ Feature: Load SEFT collection instruments
     and given survey
   So that I am assured that the right collection instrument is available for the right RU
 
-    Scenario: Load collection instrument
+  Scenario: Load collection instrument
     Given the 201803 collection exercise for the QIFDI survey has been created
     When the internal user navigates to the collection exercise details page for QIFDI 201803
     Then the user is able to load the collection instruments
+
+  Scenario: Load collection instrument wrong file type selected
+    Given the 201803 collection exercise for the QIFDI survey has been created
```

Comment:
Going forward I wonder if we need to keep defining `Given the 201803 collection exercise for the QIFDI survey has been created` as it can probably be an assumption. It would simplify and make the action step clearer in the BDD stories. It would look something like
```
Given the internal user is on the collection exercise details page for QIFDI
When the internal user selects a non xlsx file
Then the error message is displayed to the user
```

Maybe something to talk about during refinement.